src/App.js

```diff
@@ -29,9 +30,15 @@ const AppRouter = () => {
           <Route path="/dashboard">
             <Dashboard />
           </Route>
+          <Route path="/profile">
+            <Profile />
+          </Route>
           <Route path="/calendar/new">
             <NewCalendar />
           </Route>
+          <Route path={ROUTES.TEAMS.NEW}>
+            <NewTeam />
+          </Route>
         </Switch>
```

Comment:
Why this change on the routing?

Reply:
This is the new route '/teams/new' where you can create new teams we are trying to avoid magic strings so we can change those routes in the future without changing the code, take a look at:
https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings

Reply:
We need to add a task to refactor the old routes